product/views/MiqEvent.yaml

```diff
@@ -15,25 +15,31 @@ title: Events
 name: Event
 
 # Main DB table report is based on
-db: MiqEvent
+db: MiqEventDefinition
```

Comment:
Umm, it's a little confusing, the controller is `miq_event` but the model is `miq_event_definition`. Shouldn't we try to increase consistency?